tests/Unit/DataStructures/Test_ApplyMatrices.cpp

```diff
@@ -131,6 +132,26 @@ struct CheckApply<LocalScalarTag, LocalTensorTag, Dim, Dim> {
     }
     CHECK(apply_matrices(ref_matrices, get(get<LocalScalarTag>(source_data)),
                          source_mesh.extents()) == vector_result);
+
+    // Test multiple independent components in a "vector"
+    using DataType =
+        std::decay_t<decltype(get(get<LocalScalarTag>(source_data)))>;
+    const size_t num_pts = get(get<LocalScalarTag>(source_data)).size();
+    DataType t0{2 * num_pts};
+    for (size_t i = 0; i < num_pts; ++i) {
+      t0[i] = get(get<LocalScalarTag>(source_data))[i];
+      t0[i + num_pts] = t0[i];
```

Comment:
up to you: this is a linear operation so multiply the second component by  2.

Reply:
Okay, multiplied by 2